python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -345,6 +345,7 @@ def __call__(self, index):
 
     def __getitem__(self, index):
         """Return a subjob based upon index"""
+        logger.debug("__getitem__")
```

Comment:
I'd really rather not, this makes it clear in the debug when the job was potentially loaded from disk. There are similar debug messaged in the Registry for the same reason that should any of these crop up a lot in the ```--debug``` usage then this should be addressed, the message should show only a few times in normal use or on certain calls which trigger this behaviour.